set.mm

```diff
@@ -72597,6 +72668,61 @@ Power Set ( ~ ax-pow ).  (Contributed by Mario Carneiro, 20-May-2013.)
       MURHUCVFURHRUDUEUFUJUGUHUI $.
   $}
 
+  ${
+    $d A x $.  $d B x $.  $d C x $.
+    $( One way of expressing membership in the difference of domains of two
+       nested relations.  (Contributed by AV, 26-Oct-2023.) $)
+    releldmdifi $p |- ( ( Rel A /\ B C_ A ) -> ( C e. ( dom A \ dom B )
+                                    -> E. x e. ( A \ B ) ( 1st ` x ) = C ) ) $=
+      ( wrel wss wa cdm cdif wcel cv c1st wceq wrex wn eldif wb releldm2 adantr
+      cfv anbi1d syl5bb wral simprl relss 1stdm sylan eleq1 syl5ibcom rexlimdva
+      impcom con3d ralnex syl6ibr adantld imp rexdifi syl2anc ex sylbid ) BEZCB
+      FZGZDBHZCHZIJZAKZLTZDMZABNZDVEJZOZGZVIABCINZVFDVDJZVLGVCVMDVDVEPVCVOVJVLV
+      AVOVJQVBABDRSUAUBVCVMVNVCVMGVJVIOACUCZVNVCVJVLUDVCVMVPVCVLVPVJVCVLVIACNZO
+      VPVCVQVKVCVIVKACVCVGCJZGVHVEJZVIVKVCCEZVRVSVBVAVTCBUEUKVGCUFUGVHDVEUHUIUJ
+      ULVIACUMUNUOUPVIABCUQURUSUT $.
+  $}
+
+  $( The function value for the first component of an ordered pair is the
+     second component of the ordered pair.  (Contributed by AV,
+     17-Oct-2023.) $)
+  funfv1st2nd $p |- ( ( Fun F /\ X e. F )
+                      -> ( F ` ( 1st ` X ) ) = ( 2nd ` X ) ) $=
+    ( wfun wcel wa c1st cfv c2nd cop wceq wrel funrel 1st2nd sylan eleq1 adantl
+    wb wi funopfv adantr sylbid impancom mpd ) ACZBADZEBBFGZBHGZIZJZUFAGUGJZUDA
+    KUEUIALBAMNUDUIUEUJUDUIEUEUHADZUJUIUEUKQUDBUHAOPUDUKUJRUIUFUGASTUAUBUC $.
+
+  $( If the first component of an element of a function is in the domain of a
+     subset of the function, the element is a member of this subset.
+     (Contributed by AV, 27-Oct-2023.) $)
+  funelss $p |- ( ( Fun A /\ B C_ A /\ X e. A )
+                  -> ( ( 1st ` X ) e. dom B -> X e. B ) ) $=
```

Comment:
I believe this is also valid if `A` is a relation.

Reply:
No, see example for ~funeldmdif .